core/test/unit/models/post_spec.js

```diff
@@ -2,15 +2,48 @@ var should = require('should'), // jshint ignore:line
     sinon = require('sinon'),
     models = require('../../../server/models'),
     common = require('../../../server/lib/common'),
-    utils = require('../../utils'),
-
+    testUtils = require('../../utils'),
     sandbox = sinon.sandbox.create();
 
-describe('Models: Post', function () {
+describe('Unit: models/post', function () {
     before(function () {
         models.init();
     });
 
+    after(function () {
+        sandbox.restore();
+    });
+
+    describe('onValidate', function () {
+        let knexMock;
```

Comment:
@kirrg001 Travis is failing here for Node v4. Do you know why it worked for you?

Reply:
See https://github.com/TryGhost/Ghost/blob/master/core/test/unit/models/user_spec.js#L13

Reply:
nvm, found the issue 😂